sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/tests/Listeners/ScanBlobScanLogHybridPollingStrategyTests.cs

```diff
@@ -25,6 +25,7 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.Listeners
 {
+    [NonParallelizable]
```

Comment:
Why?

Reply:
I should remove. This was an experiment to see if it helped with the thread contention which was causing so much variability with tests with high thread contention.